cypress/e2e/items.cy.ts

```diff
@@ -244,4 +244,31 @@ describe('Items', () => {
     cy.findByText('PcfCM1jp0SUV').should('exist');
     cy.findByText('Zf7P8Qu8TD8c').should('exist');
   });
+
+  it('delete an item', () => {
+    cy.findAllByLabelText('Row Actions').first().click();
+    cy.findByText('Delete').click();
+
+    cy.findByText('ID: KvT2Ox7n').should('exist');
+
+    cy.startSnoopingBrowserMockedRequest();
+
+    cy.findByRole('button', { name: 'Continue' }).click();
+
+    cy.findBrowserMockedRequests({
+      method: 'DELETE',
+      url: '/v1/items/:id',
+    }).should((patchRequests) => {
+      expect(patchRequests.length).equal(1);
+      const request = patchRequests[0];
+      expect(request.url.toString()).to.contain('KvT2Ox7n');
+    });
+  });
+
+  it('should display a link to the system page when the item has a system id', () => {
```

Comment:
I think this should mention delete in some way, or be under a separate describe with the test above.